src/ngraph/codegen/compiler.cpp

```diff
@@ -185,6 +185,7 @@ void codegen::CompilerCore::initialize()
     // This is for both Eigen strong and weak inlines
     args.push_back("-mllvm");
     args.push_back("-inline-threshold=1000000");
+    // args.push_back("-fpch-instantiate-templates");
```

Comment:
This is a new Clang option available from LLVM 11 to speed up compilation when using PCH but commented out now. Will enable in a later PR.